src/controller/index.js

```diff
@@ -11,7 +11,7 @@ const compare = (obj1, obj2) => {
 };
 
 class Controller extends ENIP {
-    constructor() {
+    constructor(connectedMessaging = false) {
```

Comment:
Should we default the connection to Connected? What are your thoughts?

I think that if it is stable, then it is better as connected messaging is less constricting in terms of bandwidth.

Reply:
This depends on how we handle your last comment about the UCMM-fallback. Without fallback, I'd rather use the tried-and-tested UCMM as default, but if I implement a decent unconnected-fallback (at least during the connected process), I see no reason not to default to connected Messaging. I'd like to keep the option, though, even with the fallback.

Reply:
That makes sense, with a good fallback built into the forward open request then I think we should default to connected messaging for sure.  I definitely think that it should remain an option upon class instantiation as well (just change the `false` default to `true` lol)

Reply:
I believe this can be resolved with the comments below, we will address the fallback in another PR.